database/utilities/seed_pgs.js

```diff
@@ -0,0 +1,66 @@
+const fs = require('fs');
+const faker = require('faker');
+const path = require('path');
+const moment = require('moment');
+
+const start = Date.now();
+console.log(`Start: ${moment().format('HH:MM:SS')}`);
+
+const randomNumberGenerator = (min, max) => {
+  let randomNum = Math.floor(Math.random() * max);
+  if (randomNum < min) {
+    randomNum = Math.floor(Math.random() * max);
+  }
+  return randomNum;
+};
+
+
+const years = ['2019', '2018', '2017', '2016', '2015'];
+const months = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+];
+
+
+// const fileDestination = path.join(__dirname, 'reviews.csv');
+const reviewStream = fs.createWriteStream('reviews.csv');
```

Comment:
I have not done this either, but it might be worth git ignoring `*.csv` so that no csv files generated will be added to git commits

Reply:
Thanks Mike! it's in my gitignore but commented out at the moment